cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -316,16 +316,16 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Run command runs a compiled Ballerina program. \n");
+        out.append("Run command executes a compiled Ballerina program. \n");
```

Comment:
Here we still start with an upper case letter.

Shall we recheck all the changes to make sure the changes done are uniform across the commands?

Reply:
Changed to start with an uppercase letter inside the `printLongDesc` method in all the files under `cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/`.